RIS.js

```diff
@@ -1266,6 +1268,21 @@ function processTag(item, tagValue, risEntry) {
 				zField = ['extra'];
 			}
 		break;
+		case "H1":
+			//multiple occurences are not saved
+			if(item.libraryCatalog) {
+				item.multipleLibraries = value;
+			}
+		break;
+		case "H2":
+			//multiple occurences are not saved
+			if(item.callNumber) {
+				zField = ['notes'];
+				if(item.multipleLibraries) {
+					value = item.multipleLibraries+": "+value;
+				}
+			}
+		break;
```

Comment:
Okay. Any chance to make this translator tests language-independent? E.g. one test fails because
```
         -   "date": "June 1, 2014"
         +   "date": "Juni 1, 2014"
```